### ISSUE TITLE
Runtime Error - Index out of range

### ISSUE THREAD

**Issue by AndrewSkelton** [brentp/smoove#68]
Hi Brent, 

Thanks for your work on Smoove, and other tools, it's much appreciated! I've got the following error with Smoove and was hoping you could give me a pointer or two as to anything I can do to identifiy the cause?

Any feel for if this could be a problem with the sample coming after 42-1.bam, or with 42-1.bam itself?

Ran Using Docker:
```
docker run -v /home/dnanexus/Working/:/home/ \
                -it brentp/smoove  smoove call -x \
                                               --name SmooveTest \
                                               --exclude /home/ceph18.b37.lumpy.exclude.2014-01-15.bed \
                                               --fasta /home/hs37d5.fa \
                                               -p 16 \
                                               --genotype `ls Bams/*.bam | awk '{print "/home/"$0}' | tr '\n' ' '`
```

The following error comes through: 

```

[smoove] 2019/05/01 09:22:09 removed 5646407 alignments out of 9096282 (62.07%) with low mapq, depth > 1000, or from excluded chroms from 47-1.disc.bam in 595 seconds
[smoove] 2019/05/01 09:22:09 removed 615792 alignments out of 9096282 (6.77%) that were bad interchromosomals or flanked-splitters from 47-1.disc.bam
[smoove] 2019/05/01 09:22:10 removed 2277133 singletons and isolated interchromosomals of 2558025 reads (89.02%) from 11-1.disc.bam in 39 seconds
[smoove] 2019/05/01 09:22:10 280892 reads (3.23%) of the original 8700896 remain from 11-1.disc.bam
[smoove] 2019/05/01 09:22:12 removed 2267394 singletons and isolated interchromosomals of 3019582 reads (75.09%) from 42-1.disc.bam in 47 seconds
[smoove] 2019/05/01 09:22:12 752188 reads (8.81%) of the original 8536642 remain from 42-1.disc.bam
panic: runtime error: index out of range

goroutine 386439 [running]:
github.com/brentp/smoove/lumpy.countBases(0xc01b77f710, 0x2, 0x2, 0x7e, 0x2, 0x0, 0x72fd00)
        /home/brentp/go/src/github.com/brentp/smoove/lumpy/badsplit.go:54 +0x152
github.com/brentp/smoove/lumpy.badSplitter(0xc1011e0840, 0xdc1200)
        /home/brentp/go/src/github.com/brentp/smoove/lumpy/badsplit.go:121 +0x6e
github.com/brentp/smoove/lumpy.remove_sketchy(0xc0001a0be0, 0x11, 0x3e8, 0x7ffd9aa1cd87, 0xf, 0x7ffd9aa1cd51, 0x2d, 0xc000150480, 0x4, 0x4, ...)
        /home/brentp/go/src/github.com/brentp/smoove/lumpy/depthfilter.go:293 +0x588
github.com/brentp/smoove/lumpy.remove_sketchy_all.func1(0xc0000a5e60, 0x3e8, 0x7ffd9aa1cd87, 0xf, 0x7ffd9aa1cd51, 0x2d, 0xc000150480, 0x4, 0x4, 0x1, ...)
        /home/brentp/go/src/github.com/brentp/smoove/lumpy/depthfilter.go:373 +0x234
created by github.com/brentp/smoove/lumpy.remove_sketchy_all
        /home/brentp/go/src/github.com/brentp/smoove/lumpy/depthfilter.go:371 +0x191
```

Cheers, Andrew

**Comment by brentp** [comment]
any change you can share 42-1.disc.bam and 42-1.split.bam?
were the bams aligned with bwa mem?

**Comment by brentp** [comment]
we resolved this offline. will be out in next release.

**Comment by fangli08** [comment]
Hi, 
I have the same error. Could you please instruct how to compile the latest version? 

Best, 
Li

**Comment by brentp** [comment]
Hi, I just published 0.2.4 as the latest release, please give it a try.
